app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -457,7 +457,16 @@ function isBoltOnlyPayment()
     {
         return Mage::getStoreConfig('payment/boltpay/skip_payment');
     }
-
+    
+    /**
+     * Return disabled customer groups for Bolt
+     * @return array
```

Comment:
We can do a better job documenting that we are returning an array of customer group Id integers.